UI/css/custom_style.css

```diff
@@ -1,37 +1,68 @@
wq1hajDXVQ3HTeQt1faKvsrQIJqN2dqGgGKZmDI+MLcA0J4fMrirGcGAWEuEU1GTgxH+fxtlznYqiuL106vY/PX4fUO1G33AABTlWOGqdKGCwkL+igEEI1YfQvl5RLk3j9ppDbsjsNWclGkVi88qA2FBdXqIspljtrp4hNWJyckq0UtU9xQXYXu8GBoIaKAyi93H85q13MTQ6qVySGKmGVdU4sv9J+tN5jHt5aNTjqdkY/Hwk6rosNA1ZQjym9Lkdnc0j6G+VsJbGUezFFUK6IYyF2PWKBAyRMIm6slFRWqyG+RiMtLT3sl6GvqExDA6Po6TQwWDDgQp3Adv9dFl5GA/HUFTrwCFCdmPjSjzywN9QN+UAgK1NG/HzXz2F3Tu/BZ9HB1uaFdNkdmm5LOHycg/2vfgLWOkWpVWCicaV1dAz6FlcPiWz2qmKsRkeD2PDZyuRmWFTdGdmgjh+isG7oRAt3WMYGBxVzBooIaeTOodUZWDEpgZiDM3L7fiHv3seRTyM7XdsUtIQ371rx+cQ/xVw3869SKs0MNQUPb+6CBqWV3rUc3Xv0palrC/t+4Qvnh83pGyUL4qb6iqRmvSX8zF0/v5jFFdloOf9XgwMCay0UlFGQ2TPhTcUpQsiKqj3Jlpi5M5jx1f34d0PWxRdYUBUTxKKH738VUxeGEWajhmPNF4WqCJ6BVKTS13z91NKlrSIOJPySzNYv7qagXqmWmBoRIKPODLSU/g7AJF0siyvdCPPk4GxnjGkOkzQUwkDhGd+YQaGuybxxDM/xIFn/1FlYMKwWPrdOz+PSzR03/jaCyhdXYRxHtLiIqp0obsf751shcWssSMwXrm8EqtXVChPkBz//2BWR8MgepQBp8Ou6IUZar536izrAzjXL02d6Py4D4GZENKpdwUMGysLcjF2egiGfC04kL1P0a86yjNxrrUHm5qewdvHvoHbNq1DnIeRlpqCB++9C6daLuDIy3+mfdAiLrVg4tVx4SIe3v0Ffm3mI26qD98/8B2sYmZkTLhINl47gpKOGxbGFT3U16rbliErS5OqWD+93oi125pw520NWLf1IXoKncqihJ7BaOCJe1gjFjnWSE5T6cMlgY+SMVcR6cz40fTkczh1pl3BViRcUuTAYw/cxUiSKd14RIvHhWCiKF+Paixbkwn3KvHnhWTyag399JI1UzIjPjStqUBRgRgewpp+djcT5i/cvpEb1dMtzSt3lJut5Z1mJtq1VW6OZPZDiYaIjOkpWnMVVsUxZTWhpN6OvuZu/D0hfWj/HlSVaVZ+002rce9d9Xj1xddV1qMWTL4UtA2Y5cFGlE5fkfEkxv2FzC7VExWqWOTkvKgsK1T6qnSMp1nInJcWJ7mNhV/pl7Bvw5plbAuq6xVbihnBVJtyFTpK2BqLYiwYhbu+AKePncArmxvwzJMP8RANSKMaVNCXK0dLWoscrkKJHKDAXoITDTn8uaJcLesrBqhPEllcZOO8EGKTE3Za12SRq5TA9MzSh2lehA5ezeFAe04230W4dG4Qf3vnRlx48zmcf+N76Pz1v+OJL21BqNOvkTM78Pa7zbhwcUB9y5KFBZJo8DZr6XaUBdcmyfsKwVzu0JL3Rd9LqjItTPeQ684gdDSHLwPmooSevxW1n92E8kQwIdHU60f/iD3f+xmKJX7V6ema4mju9+EHX7sP939pK1IpSZPZgtt31+D4z08yxU2Hm5FVsuTlir5FtcAn14TuUT+8vmnVLXsRZKiiJKtVEw2LP65bvyaMU2kwrDYjMlOMGBmYQdM2D2Gk3VQIJUnwgV7UVRSiZtHd1UXeNPR++AZ6beuZHNBSp/KAgp1oObcOX7wzopjNJINb1i0js4ybCTlJ38R9CPy0lNBA2PKgLHKNc3nfUo1GE1GU0tHLfZ8kzcWjrmJWEuautnFavYvo7JUsxoe7t95Ct6FlNJJrdvddUjSchFV6miTy5Ik3k15e6QC1qKjOwnQkpvxeb0cZ+odGFZRlXDpdSW2V6J6EvXrFaLKuSTkAv9+GWE8/ljVUK3cl/SLZwaERvsPKuEnbQrlK0gs9SyqK2UUHSKiY8ZXHtvAOqg5WRkW3MEW747aNC5NEJ3974iN+r0BZqZtORGbrVAx8vkucqxl+MiqPhbbCkGnEkbZ+7PUFUOjU9Ds7R37zMTo2AR91Wm4LpWxjTPzAV+7Gy699gIKbGvD4g9sJc+3aJhgOo+vjixzF9ZRkF+9aTb/hS5Ns4mQESlmE2bNPPwq5TbTwMtrC5Hwxato7u/HqgfdJ2I56RigS80rxkpmuASIix6wSbRGFQDSD90ZTXQFMTHoTx8I7ZpsNaQ1bcPC1P2PXF7fi1o2rFaRFZ7+z53Em/XfAac9mvOte8KnH/3QSrx5t42RJ3pfecGhsi+wvFzGISaOoWvnNnfJaM6H4Aiu5UJa8MouBvbpBoD6J8Zmlvnxwug1PP/sC5/qR3Zi/5K7Z6/Oj851u5NotDBA02MW4gCWFAXxghDHyiIKiLCxXq4828VZx8BgOHT6qbhWTVzJF+XZ8blOjuhdOGqTm1g78549/yVCLoWe6Se3HnAgNhZ6Rrklcz2WhMGAhT+KykkUMrBFZenR0DyAvjxfYi/4RkEEiXT9TtjAvzs5Tok/tO8o0Z5KeIwNVhXa0tl9USJDDaD7LTEeCBRKV5FyKnOycBNEsp878LyorSlX4Jvdz86p9HX687zimGU7u3LYZHrcL+cyMLBYjgvxLZHRsktDtwcGXj+C9I2fhWmXnvwQzCvot5z9e+Eegu1dsSCYiTOy1tS3oGxxBM+/BRN4iwMHhCeiu9V+PbFXSrDAjnK42BvZjg2yZRU61CyaLGT7qY0F2GlIJR2FU9HbSR//Kv0iS/xVxgirSL5dlWek2FDqyEKHY5Swk753gk0uLP3CGB4hJOG9ejc01LiKK91PTYbxJXxw52U7VyEJhSYZaV/7N+bT/9fwfoeSNOue3zr0AAAAASUVORK5CYII=');
```

Comment:
I need the PSD someplace - it is a template to produce the headers needed, of which we will eventually need quite a few. If I am unavailable, it will be available to my successor because it is in the source.